scene/resources/resource_format_text.cpp

```diff
@@ -602,9 +602,11 @@ Error ResourceLoaderText::load() {
 		}
 
 		int_resources[id] = res; //always assign int resources
-		if (do_assign && cache_mode != ResourceFormatLoader::CACHE_MODE_IGNORE) {
-			res->set_path(path, cache_mode == ResourceFormatLoader::CACHE_MODE_REPLACE);
-			res->set_scene_unique_id(id);
+		if (do_assign) {
+			if (cache_mode != ResourceFormatLoader::CACHE_MODE_IGNORE) {
+				res->set_path(path, cache_mode == ResourceFormatLoader::CACHE_MODE_REPLACE);
+			}
+			res->set_scene_unique_id(id); // FIXME: Temporary fix for resetting internals ids. Revert this block after proper fix.
```

Comment:
I think this is still correct, even if a temporary fix, likely does not need the comment

Reply:
It won't be necessary after the issue is fixed though.